src/datasets/experiment_monitoring.md

```diff
@@ -0,0 +1,60 @@
+# Experiment monitoring datasets
+
+<!-- toc -->
+
+Experiment monitoring datasets are designed to power dashboards, such as [Experiment Enrollment Grafana dashboard](https://grafana.telemetry.mozilla.org/d/XspgvdxZz/experiment-enrollment?orgId=1), for monitoring experiments in real time. Currently, datasets for monitoring the number or enrollments and number of searches performed by clients enrolled in experiments are available.
+
+## Experiment enrollment data
+
+`moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live` provides enrollment, unenrollment, graduate, update and failure aggregates for experiments and branches over 5-minute intervals. This live view is also the basis of several derived tables:
+
+| Dataset name                                                                             | Description                                                         |
+| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| `moz-fx-data-shared-prod.telemetry.experiment_unenrollment_overall`                      | Overall number of clients that unenrolled from experiments          |
```

Comment:
```suggestion
| `mozdata.telemetry.experiment_unenrollment_overall`                      | Overall number of clients that unenrolled from experiments          |
```

Can we consistently list all user-facing views in mozdata rather than shared-prod? I'd expect that we give only the live view path in shared-prod.